src/PatientChecking/PatientChecking/Controllers/PatientController.cs

```diff
@@ -1,16 +1,59 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using PatientChecking.Services.Repository;
+using PatientChecking.Services.ServiceModels;
+using PatientChecking.Services.ServiceModels.Enum;
+using PatientChecking.Views.ViewModels;
 using System;
 using System.Collections.Generic;
+using System.Dynamic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace PatientChecking.Controllers
 {
     public class PatientController : BaseController
     {
-        public IActionResult Index()
+
+        private readonly IPatientService _patientService;
+
+        public PatientController(IPatientService patientService)
         {
-            return View();
+            _patientService = patientService;
+        }
+
+        public IActionResult Index(int sortOption = (int)PatientSortSelection.ID, int pagingOption = 10, int currentPage = 1)
+        {
+            PagingRequest request = new PagingRequest()
+            {
+                PageIndex = currentPage,
+                PageSize = pagingOption,
+                SortSelection = sortOption
+            };
+            PatientList result = _patientService.GetListPatientPaging(request);
+            List<PatientViewModel> patientsVm = new List<PatientViewModel>();
+            foreach (Patient p in result.Patients)
```

Comment:
Should add empty line above for readability. Please check for other places.

Reply:
empty line has been added. Please check this for me again.